index.html

```diff
@@ -31,7 +31,139 @@ <h1>I’m Mariane
      <li><a href=""><img src="images/vector2.svg" alt=""></a></li>
     </ul>
    </section>
+   <section>
+    <article class="snapshoot"><img src="images/Snapshoot.png" alt=""></article>
+    <article>
+     <h1>Tonic</h1>
+     <div class="frame">
+      <h2>CANOPY</h2>
+      <div></div>
+      <p >Back End Dev</p>
+      <div></div>
+      <p>2015</p>
+     </div>
+     <div class="text">
+      <p>A daily selection 
```

Comment:
- [ ] To keep your clean code, you should consider removing this `<div>` wrapping your paragraph. You can use the `class="text"` inside your `<p>` tag, for styling purposes.